.github/workflows/release-desktop.yml

```diff
@@ -2,7 +2,7 @@ name: Release desktop wallet
 
 on:
   push:
-    tags: ['alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+*']
+    tags: ['alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+', 'alephium-desktop-wallet@[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+']
```

Comment:
Nit: I guess both patterns could be merged to one by setting the last part as an optional group (to be tested):

`alephium-desktop-wallet@[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?`

Reply:
pushed